public/js/plugins/bootstrap-input-spinner.js

```diff
@@ -4,330 +4,325 @@
+                if (disabled || readonly) {
+                    resetTimer();
+                }
+                var originalClass = $original.prop('class');
+                var groupClass = '';
+                // sizing
+                if (/form-control-sm/g.test(originalClass)) {
+                    groupClass = 'input-group-sm';
+                } else if (/form-control-lg/g.test(originalClass)) {
+                    groupClass = 'input-group-lg';
+                }
+                var inputClass = originalClass.replace(/form-control(-(sm|lg))?/g, '');
+                $inputGroup.prop(
+                    'class',
+                    `input-group ${groupClass} ${config.groupClass}`,
+                );
+                $input.prop('class', `form-control ${inputClass}`);
 
-      function parseLocaleNumber(stringNumber) {
-        var numberFormat = new Intl.NumberFormat(locale);
-        var thousandSeparator = numberFormat.format(1111).replace(/1/g, "");
-        var decimalSeparator = numberFormat.format(1.1).replace(/1/g, "");
-        return parseFloat(
-          stringNumber
-            .replace(new RegExp("\\" + thousandSeparator, "g"), "")
-            .replace(new RegExp("\\" + decimalSeparator), ".")
-        );
-      }
-    });
+                // update the main attributes
+                min = parseFloat($original.prop('min')) || 0;
+                max = isNaN($original.prop('max')) || $original.prop('max') === ''
+                    ? Infinity
+                    : parseFloat($original.prop('max'));
+                step = parseFloat($original.prop('step')) || 1;
+                stepMax = parseInt($original.attr('data-step-max')) || 0;
+                var newDecimals = parseInt($original.attr('data-decimals')) || 0;
+                var newDigitGrouping = !(
+                    $original.attr('data-digit-grouping') === 'false'
+                );
+                if (decimals !== newDecimals || digitGrouping !== newDigitGrouping) {
+                    decimals = newDecimals;
+                    digitGrouping = newDigitGrouping;
+                    numberFormat = new Intl.NumberFormat(locale, {
+                        minimumFractionDigits: decimals,
+                        maximumFractionDigits: decimals,
+                        useGrouping: digitGrouping,
+                    });
+                }
+            }
 
-    return this;
-  };
+            function parseLocaleNumber(stringNumber) {
+                var numberFormat = new Intl.NumberFormat(locale);
+                var thousandSeparator = numberFormat.format(1111).replace(/1/g, '');
+                var decimalSeparator = numberFormat.format(1.1).replace(/1/g, '');
+                return parseFloat(
+                    stringNumber
+                        .replace(new RegExp(`\\${thousandSeparator}`, 'g'), '')
+                        .replace(new RegExp(`\\${decimalSeparator}`), '.'),
+                );
+            }
```

Comment:
this function has been working as intended for our current use cases, and altering it might introduce unforeseen issues. I'll leave it as-is for now, but we can revisit if locale-specific parsing problems arise